app/controllers/application.js

```diff
@@ -3,7 +3,7 @@ import { infoLinks } from 'ember-styleguide/constants/es-footer';
 import styleguideLinks from 'ember-styleguide/constants/links';
 import replaceLinks from 'ember-website/utils/replace-links';
 
-export default Controller.extend({
-  links: replaceLinks(styleguideLinks),
-  infoLinks: replaceLinks(infoLinks),
-});
+export default class Application extends Controller {
```

Comment:
Can you update this line (similarly, in other controllers) by adding the suffix `Controller`? I'd like the code to follow what an Ember CLI-generated file would show.

```suggestion
export default class ApplicationController extends Controller {
```

Reply:
Interesting, I asked my team about this a few months ago and the consensus was that the prefix was redundant. 🤷